### DISCUSSION TITLE
Fusion Reactor Connectivity

### DISCUSSION THREAD

**Discussion by NiclasgamePoint** [MikaylaFischler/cc-mek-scada#278] [Support] [answered]
is it possible to connect  a fusion reaktor to the system?

**Comment by MikaylaFischler** [comment, marked as answer]
Since fusion reactors don't have any safety issues like the fission reactors and are simpler to operate once started, it's not currently a feature. However, I'll be adding some integration with them in the future, likely after the full release of this.

**Reply by NiclasgamePoint** [reply]
thanks, I'll forward to it

**Reply by JakeTheOctoling** [reply]
Let us know when that patch comes out. I have five fusion reactors at play with a space base and I found that apparently if the injection rate is too high then it will explode. The max I found is 99 (which this number is known to explode according to in game experience) but I do have an induction matrix that has a very large capacity that the game says infinite on, I would like to see if this program could also show a battery percentage and capacity. I might have to figure out ways to get more generators running but I'll find a way.

**Reply by MikaylaFischler** [reply]
99 is indeed the limit for Mekanism fusion reactors, I didn't know you could even set one higher than that. I was not aware of the fusion reactors exploding under any circumstances unless something changed since I last looked, but I've never heard of or tried setting an injection rate above 99. As a heads up, the maximum number of fusion reactors a given system will let you add is going to be 4, as is done with fission, and the fusion reactors will subtract from the total 4 reactors of any type. The existing system already shows induction matrix charge, capacity, etc information, so there won't be anything new with that.

I have a queue of feature requests, and this near the back. The associated issue #527 will be closed when this feature is implemented and it will be mentioned in the release that contains it. I will 100% forget about this thread as I already had, since the issues are what are used for tracking features.
